### DISCUSSION TITLE
Infinity plugin re-install challenge

### DISCUSSION THREAD

**Discussion by egliptor** [grafana/grafana-infinity-datasource#63] [Q&A] [answered]
Hi,

I uninstalled the Infinity plugin from my Linux machine to see if the version of Grafana and Infinity were causing issues.

Infinity: version 0.6.0
Granfana: version 7.5.4
Operating System: Linux Manjaro

Grafana was reinstalled, however when trying to re-install Infinity I keep getting an 404 error like the plugin's name has changed

![Plugin Re-install Challenge](https://user-images.githubusercontent.com/73836983/116094610-9719e680-a69f-11eb-866c-28c4f0141236.png)

Has something changed to install the plugin?

Kind regard

**Comment by yesoreyeram** [comment, marked as answer]
Infinity datasource is not yet published to grafana.com / grafana-cli. So you need to pass the plugin url when installing through grafana cli.

```sh
grafana-cli --pluginUrl https://github.com/yesoreyeram/grafana-infinity-datasource/releases/download/v0.6.0/yesoreyeram-infinity-datasource-0.6.0.zip plugins install yesoreyeram-infinity-datasource

```

**Reply by yesoreyeram** [reply]
Infinity plugin is now available in [grafana.com plugins page](https://grafana.com/grafana/plugins/yesoreyeram-infinity-datasource/). You can install from there into your instance of grafana. so now you can install latest published version by running

```sh
grafana-cli  plugins install yesoreyeram-infinity-datasource

```

**Comment by egliptor** [comment]
Thanks, re-install completed
